_posts/newsletters/2024-01-24-newsletter.md

```diff
@@ -0,0 +1,382 @@
+
+The majority of the grant from the Sloan Foundation is budgeted to pay for our new Executive Director and Community Manager. Besides that we've budgeted money to support staff travel, the US-RSE conference, fund community activities through our new Community and Travel Fund program, and fund smaller items such as code of conduct trainings for our code of conduct committee members. CI charges 10% of our income for providing resources, infrastructure, and support (trust us when we say we make good use of those resources). Switching from OCF to CI turned out to been an unexpectedly complex change and it has been a very steep learning curve. We appreciate our members' patience as we navigate the world of non-profit regulations and guidelines.  
+
+
+
+<a name="involved"></a>
+# **Get Involved**
+
+There are lots of ways to get involved with the US-RSE community. Of course, you can [join us on Slack](https://us-rse.org/join). We’re also looking for ideas and help in many other places. See our [list of projects](https://docs.google.com/document/d/1jjVD0WkeeWZJI6yqSKyMdIjtClzolsxv75RkpLju17I/edit?usp=sharing) and let us know how you’d like to help. Help with things like the Website Committee, Social Media, Community Engagement, Fundraising, and Newsletter are all needed and welcome.
+
+New guidance was recently released on how to create new [Affinity Groups](https://us-rse.org/wg/group-management/form-an-affinity-group) and [Working Groups](https://us-rse.org/wg/group-management/form-a-working-group). If you see an unmet need in the US-RSE community, we encourage you to create your own group!
+
+As always we thank everyone who is already contributing for their help in shaping and growing the organization.
+
+[US-RSE Working Groups](https://us-rse.org/working-groups/):
+
+* [Diversity, Equity, and Inclusion](https://us-rse.org/wg/dei/)
+* [Education & Training](https://us-rse.org/wg/education_training/)
+* [Group Management](https://us-rse.org/wg/group-management/)
+* [Outreach](https://us-rse.org/wg/outreach/)
+* [RSE Empowerment in National Labs](https://us-rse.org/wg/rse-empowerment-national-labs/)
+* [Website](https://us-rse.org/wg/website/)
+* [Code Review](https://us-rse.org/wg/code-review/)
+* [User Experience](https://us-rse.org/wg/ux/)
+* [Mentorship Program](https://us-rse.org/wg/mentorship-program/)
+
+
+<a name="jobs"></a>
+# **Recent Job Postings**
```

Comment:
there are 2 more recent posts on the board

Reply:
Thank you for letting me know! I will add these.

Reply:
Thanks! We fixed/changed in latest commit!